packages/babel-plugin-proposal-private-methods/test/fixtures/accessors/set-only-getter/output.js

```diff
@@ -14,7 +14,7 @@ class Cl {
       value: 0
     });
 
-    babelHelpers.readOnlyError("#privateFieldValue");
+    this, 1, babelHelpers.readOnlyError("#privateFieldValue");
```

Comment:
This is a few more unnecessary bytes, but this path will always result in an error so it's highly improbable that it will happen in production code.